File: app/src/main/java/com/letianpai/robot/desktop/parser/RobotMenu.java
package com.letianpai.robot.desktop.parser;

/**
 *
 */
public class RobotMenu {
    private int code;
    private String msg;
    private RobotMenuData data;

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

    public RobotMenuData getData() {
        return data;
    }

    public void setData(RobotMenuData data) {
        this.data = data;
    }

    @Override
    public String toString() {
        return "{" +
                "code:" + code +
                ", msg:'" + msg + '\'' +
                ", data:" + data +
                '}';
    }
}

File: app/src/main/java/com/letianpai/robot/desktop/parser/RobotMenu.kt
package com.letianpai.robot.desktop.parser

/**
 *
 */
class RobotMenu {
    var code: Int = 0
    var msg: String? = null
    var data: RobotMenuData? = null

    override fun toString(): String {
        return "{" +
                "code:" + code +
                ", msg:'" + msg + '\'' +
                ", data:" + data +
                '}'
    }
}
